lib/R.sc

```diff
@@ -1277,36 +1283,50 @@ RSineOscillatorModule : RModule {
 	*params {
 		^[
 			'Range' -> (
-				Spec: ControlSpec.new(-2, 2, 'lin', 1, 0),
+				Spec: ControlSpec.new(-8, 8, nil, 1, 0),
 				LagTime: 0.01
 			),
 			'Tune' -> (
-				Spec: ControlSpec.new(-600, 600, 'lin', 0, 0, "cents"),
+				Spec: ControlSpec.new(-600, 600, nil, 0, 0, "cents"),
```

Comment:
unintented commits?

Reply:
`lin` is the same mistake as before but changing -2/2 to -8/8 is part of my changelog for the PR above - I wanted to give SinceOsc a higher range control since it's a convenient way to define carrier / modulator ratios.

I set the limit to 8 here but actually on a DX7 or something these actually go up to 24 or 25 but in testing I found above something like 8 the higher oscillator would drop out, or something, not sure what was happening but this range proved stable for my uses